app/models/events/pull_request.rb

```diff
@@ -24,7 +24,7 @@
 #
 module Events
   class PullRequest < ApplicationRecord
-    enum state: { opened: 'opened', closed: 'closed' }
```

Comment:
Why are we rolling back on this?

Reply:
Because we receive `open` instead of `opened` from GitHub, and it detects it as an invalid state.